cmd/minikube/cmd/root.go

```diff
@@ -50,13 +50,14 @@ var RootCmd = &cobra.Command{
 	Short: "Minikube is a tool for managing local Kubernetes clusters.",
 	Long:  `Minikube is a CLI tool that provisions and manages single-node Kubernetes clusters optimized for development workflows.`,
 	PersistentPreRun: func(cmd *cobra.Command, args []string) {
+		setFlagsUsingViper()
```

Comment:
The "version" command currently overrides this persistentprerun. It's not a big deal since that doesn't take any flags, but we might want to switch it so it doesn't override and uses a different method to disable version checking.